include/game.h

```diff
@@ -19,4 +19,33 @@ extern s32 D_800EC924;
 extern s32* D_800EC928;
 extern s32 func_800B9A5C;
 extern s32 func_800B9AC8;
-extern s32 D_800F6D08;
\ No newline at end of file
+extern s32 D_800F6D08;
+extern void func_80059CCC();
+extern void func_80059E58();
+extern void func_80099AFC();
+extern s32 D_800CDF5C;
```

Comment:
Needs to be redeclared as:
```suggestion
extern s32 D_800CDF5C[];